tests/fixtures/system.py

```diff
@@ -0,0 +1,71 @@
+#  Copyright 2019 XLAB d.o.o.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+from redfish_client import Connector
+from redfish_client.resource import Resource
+
+
+@pytest.fixture
+def system():
```

Comment:
This doesn't need to be a fixture, can you please store it in a variable instead, e.g. 

```python
SYSTEM_LENOVO = { ... }
```